### PR TITLE
Refactoring opencv_warco to enable build from install and devel

### PR DESCRIPTION
**Pull request by cdondrup**

### CODE REVIEW

File: opencv_warco/package.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
   <!-- Commonly used license strings: -->
   <license>MIT</license>
 
-  <url type="website">https://github.com/lucasb-eyer/opencv-warco</url>
+  <url type="website">https://github.com/lucasb-eyer/opencv_warco</url>
```
Comment (lucasb-eyer) [inline opencv_warco/package.xml:13]: I don't think this changes^^

**Comment by lucasb-eyer** [comment]
Thanks for doing this!

File: opencv_warco/package.xml (hunk 1 above)
Comment (cdondrup) [inline opencv_warco/package.xml:13]: Damn... grep and sed have a mind on their own ;)
Will change in the next iteration, if I don't forget.

Comment (cdondrup) [inline opencv_warco/package.xml:13]: Also the executable targets shouldn't have been renamed... It's getting late; I will attribute it to that. I hope that doesn't cause any problems.

**Pull request merged**